lib_trees/data/gm4_trees/functions/sapling/ray_fuzzy.mcfunction

```diff
@@ -4,12 +4,12 @@
 # run from gm4_trees:sapling/place_sapling
 
 # check adjacent blocks
-execute if score $found gm4_tree_data matches 0 store success score $found gm4_tree_data align xyz positioned ~0.5 ~1.5 ~0.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
-execute if score $found gm4_tree_data matches 0 store success score $found gm4_tree_data align xyz positioned ~0.5 ~-0.5 ~0.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
-execute if score $found gm4_tree_data matches 0 store success score $found gm4_tree_data align xyz positioned ~1.5 ~0.5 ~0.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
-execute if score $found gm4_tree_data matches 0 store success score $found gm4_tree_data align xyz positioned ~-0.5 ~0.5 ~0.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
-execute if score $found gm4_tree_data matches 0 store success score $found gm4_tree_data align xyz positioned ~0.5 ~0.5 ~1.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
-execute if score $found gm4_tree_data matches 0 store success score $found gm4_tree_data align xyz positioned ~0.5 ~0.5 ~-0.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
+execute if score $found gm4_tree_data matches 0 align xyz positioned ~0.5 ~1.5 ~0.5 if block ~ ~ ~ #minecraft:saplings unless entity @e[tag=smithed.block,distance=..0.5,limit=1] run function gm4_trees:sapling/summon_marker
```

Comment:
The score should be reset to 0 at the start of this function

Reply:
this function only runs if `$found` is already 0, from `place_sapling`: `execute if score $found gm4_tree_data matches 0 anchored eyes positioned ^ ^ ^ run function gm4_trees:sapling/ray_fuzzy`